transports/rabbitmq/routing-topology_mandatory_rabbit_[5,).partial.md

```diff
@@ -1 +1 @@
-The current recommendation, and the default prior to Version 5, is the `ConventionalRoutingTopology`, which relies on [fanout exchanges](https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchange-fanout) to route messages. 
+The current recommended routing topology is the `ConventionalRoutingTopology`. It was the default topology prior to Version 5. This topology relies on [fanout exchanges](https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchange-fanout) to route messages. 
```

Comment:
I think "current" needs to be changed to "currently".

However, I'm wondering if that should just be dropped from the sentence altogether instead. It's implicitly the current recommendation by us stating it here.